components/build-service/overlays/dev/apiexport.yaml

```diff
@@ -25,3 +25,6 @@ spec:
   - group: appstudio.redhat.com
     identityHash: application-api
     resource: applications
+  - group: appstudio.redhat.com
+    identityHash: application-api
+    resource: applicationsnapshots
```

Comment:
should be added here -> https://github.com/redhat-appstudio/build-service/blob/main/config/kcp/build-service-apiexport.yaml then it will create PR here